cogs/fun_channel_checks.py

```diff
@@ -43,7 +43,16 @@ async def on_message(self, message):
                         break
                     await message.delete()
                     return
-
+            if message.content.lower() == "the end":
+                messages = await channel.history(limit=200)
+                fetch_message = messages.find(lambda m: m.author == client.user)
```

Comment:
Also, this should be `lambda m: m.author == self.client.user` as you're doing this in a cog

Reply:
o ok